src/main/java/com/denizenscript/depenizen/bukkit/commands/worldedit/WorldEditCommand.java

```diff
@@ -0,0 +1,370 @@
+package com.denizenscript.depenizen.bukkit.commands.worldedit;
+
+import com.denizenscript.denizen.objects.CuboidTag;
+import com.denizenscript.denizen.objects.LocationTag;
+import com.denizenscript.denizen.objects.PlayerTag;
+import com.denizenscript.denizen.utilities.Utilities;
+import com.denizenscript.denizen.utilities.debugging.Debug;
+import com.denizenscript.denizencore.exceptions.InvalidArgumentsException;
+import com.denizenscript.denizencore.objects.Argument;
+import com.denizenscript.denizencore.objects.core.ElementTag;
+import com.denizenscript.denizencore.scripts.ScriptEntry;
+import com.denizenscript.denizencore.scripts.commands.AbstractCommand;
+import com.sk89q.worldedit.EditSession;
+import com.sk89q.worldedit.LocalSession;
+import com.sk89q.worldedit.WorldEdit;
+import com.sk89q.worldedit.WorldEditException;
+import com.sk89q.worldedit.bukkit.BukkitAdapter;
+import com.sk89q.worldedit.bukkit.BukkitWorld;
+import com.sk89q.worldedit.extension.input.ParserContext;
+import com.sk89q.worldedit.extension.platform.Actor;
+import com.sk89q.worldedit.extent.clipboard.BlockArrayClipboard;
+import com.sk89q.worldedit.extent.clipboard.Clipboard;
+import com.sk89q.worldedit.extent.clipboard.io.*;
+import com.sk89q.worldedit.function.operation.ForwardExtentCopy;
+import com.sk89q.worldedit.function.operation.Operation;
+import com.sk89q.worldedit.function.operation.Operations;
+import com.sk89q.worldedit.math.BlockVector3;
+import com.sk89q.worldedit.math.transform.AffineTransform;
+import com.sk89q.worldedit.regions.CuboidRegion;
+import com.sk89q.worldedit.session.ClipboardHolder;
+import com.sk89q.worldedit.util.io.Closer;
+import com.sk89q.worldedit.world.World;
+import org.bukkit.Bukkit;
+import org.bukkit.entity.Player;
+
+import java.io.*;
+import java.net.URLDecoder;
+
+public class WorldEditCommand extends AbstractCommand {
+    // <--[command]
+    // @Name worldedit
+    // @Syntax worldedit [cuboidschematic/cuboidclipboard/schematicpaste/schematicclipboard] (file:<file path>) (undoable) (noair) (cuboid:<CuboidTag>) (rotate:<amount>) (target:<player>)
+    // @Group Depenizen
+    // @Plugin Depenizen, WorldEdit
+    // @Required 2
+    // @Short Allows access to various WorldEdit actions.
+
+    // @Description
+    // Allows access to various WorldEdit actions.
+    // Actions can be cuboidschematic, cuboidclipboard, schematicpaste, schematicclipboard
+    // If no targets are specified, the target will be the player in the queue.
```

Comment:
This line implies the input is a list, which it isn't. If there isn't a list, the input doesn't need to exist tbh (just use `player:` arg)